src/main/java/com/aws/greengrass/certificatemanager/certificate/ServerCertificateGenerator.java

```diff
@@ -60,7 +60,10 @@ public synchronized void generateCertificate(Supplier<List<String>> connectivity
                 .kv("subject", subject)
                 .kv("reason", reason)
                 .kv("connectivityInfo", connectivityInfo)
+                .kv("previousCertExpiry", certificate == null ? "N/A" : getExpiryTime())
+                .kv("previousCertValiditySeconds", certificate == null ? "N/A" : getValidity().getSeconds())
```

Comment:
Nit: I find a little confusing that we log this information twice. IMO the generator class should log stuff about the new cert. Details about why could be part of `reason` which is provided by the monitor (e.g. the old cert expired). Maybe remove this and just log after the new cert is generated?